frontend/server/src/main/java/org/pytorch/serve/http/InferenceRequestHandler.java

```diff
@@ -111,9 +128,28 @@ private void handlePredictions(
         if (segments.length == 4) {
             modelVersion = segments[3];
         }
+        req.headers().add("explain", "False");
+        if (explain) {
+            req.headers().add("explain", "True");
```

Comment:
@abishekchiffon In case of explain, will there be two values for the header added (from line 131 and 133)?

Reply:
The key will be overridden with new values.